orcid-core/src/main/java/org/orcid/core/web/filters/AcceptFilter.java

```diff
@@ -53,10 +60,13 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
             if (isValidAcceptType(contentType))
                 requestWrapper = new AcceptHeaderRequestWrapper(request, contentType);
             else
-                if (OrcidUrlManager.getPathWithoutContextPath(request).startsWith("/oauth/"))
+                if (OrcidUrlManager.getPathWithoutContextPath(request).startsWith("/oauth/")) {
```

Comment:
Hey, 
* In what case is path null? 2) 
* why doesn't getPathWithoutContextPath without context work (that a leading questions)?